docs/config.rst

```diff
@@ -177,11 +177,39 @@ These are the available audio configurations. For specific use cases, see
 
     Expects an integer above 0.
 
-    Sets the buffer size of the GStreamer queue. If you experience buffering
+    Sets the maximum buffer size of the GStreamer queue. If you experience buffering
     before track changes, it may help to increase this, possibly by at least a
     few seconds. The default is letting GStreamer decide the size, which at the
     time of this writing is 1000.
 
+.. confval:: audio/min_buffer_size
+
+    The minimum buffer size that is given to GStreamer playbin, in bytes.
+
+    Expects an integer 0 or higher.
+
+    Sets the minimum buffer size of the GStreamer playbin. If you experience
+    buffering/stuttering, it may help to increase this. If you experience
+    latency when changing tracks, lower this value.
+    A value of 0 seems to work fine, then the minimum buffer size seems to be
+    determined by min_buffer_duration.
```

Comment:
I think this sentence is a bit confusing, especially since the default value isn't zero.